R/pkg/R/sparkR.R

```diff
@@ -31,20 +31,27 @@ connExists <- function(env) {
 #' Stop the Spark context.
 #'
 #' Also terminates the backend this R session is connected to
+#' @export
 sparkR.stop <- function() {
+  .Deprecated("sparkR.session.stop")
+  sparkR.session.stop()
```

Comment:
Yeah - while matching `init` and `stop` is good I think having `sparkR.session` and `sparkR.stop` would also be fine. 
Also we can have both `sparkR.stop` and `sparkR.session.stop` work for now (we can just remove the deprecation warning) and deprecate one of them based on user feedback ?